Editor/UI/Windows/CourseWindowFixedOverlay.cs

```diff
@@ -0,0 +1,50 @@
+/* Copyright (C) Innoactive GmbH - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Innoactive GmbH, November 2020
+ */
+
```

Comment:
```suggestion
/* Copyright (C) Innoactive GmbH - All Rights Reserved
 * Unauthorized copying of this file, via any medium is strictly prohibited
 * Proprietary and confidential
 * Innoactive GmbH, November 2020
 */

```

Not needed. This is the core and thus free version of the Creator.